api/server.js

```diff
@@ -20,14 +16,88 @@ mongoose.connection.once("open", () => {
   console.log("MongoDB connected");
 });
 
-app.use(
-  "/graphql",
-  graphql({
-    schema,
-    graphiql: true
-  })
-);
+const Location = require("./models/location");
+const Venue = require("./models/venue");
 
-app.listen(5000, () => {
-  console.log("api running at http://localhost:5000/graphql");
-});
+// GraphQl type definitions(similair to models for db)
+const typeDefs = gql`
+  type Query {
+    getLocation(id: ID): Location!
+    getLocations: [Location]
+    getVenue(id: ID): Venue
+    getVenues: [Venue]
+  }
+
+  type Location {
+    name: String!
+    state: String!
+    id: ID!
+    venues: [Venue]
+  }
+
+  type Venue {
+    id: ID!
+    businessName: String!
+    category: String!
+    website: String!
+    phone: String!
+    address: String!
+    locationId: ID!
+    location: Location
```

Comment:
What’s the difference between address, location, and locationId?

Reply:
address would be street address
locationId is the id of the location(city)
location would pull in the info about that city including other business' if its part of the query